EventProc/src/TrigScintTrackProducer.cxx

```diff
@@ -0,0 +1,365 @@
+/**
+ * @file TrigScintTrackProducer.cxx
+ * @brief 
+ * @author Lene Kristian Bryngemark, Stanford University 
+ */
+
+#include "EventProc/TrigScintTrackProducer.h"
+
+
+#include <iterator>     // std::next
+#include <map> 
+
+
+namespace ldmx {
+  
+	void TrigScintTrackProducer::configure(Parameters& ps) {
+
+	  maxDelta_                = ps.getParameter< double >("delta_max");  // max distance to consider adding in a cluster to track
+	  seeding_collection_      = ps.getParameter< std::string >("seeding_collection"); //probably tagger pad, "TriggerPadTagClusters" 
+	  input_collections_       = ps.getParameter< std::vector < std::string > >("further_input_collections"); // {"TriggerPadUpClusters" , "TriggerPadDnClusters" }
+	  output_collection_       = ps.getParameter< std::string >("output_collection"); 
+	  passName_                = ps.getParameter< std::string >("input_pass_name"); 
+	  verbose_                 = ps.getParameter< int >("verbosity");
+
+	  // TO DO: allow any number of input collections
+    
+
+	  if (verbose_)
```

Comment:
Why do you need a check for verbose if you are already setting the log level to info? You can always change the log level to something higher if you want these messages to stop.

Reply:
So to my understanding, the logger verbosity level is a global feature (so if I set it to debug, I get debug level messages from all invoked producers). For my use of clustering and tracking, which is still to work out the best parameters, it's very convenient to be able to just make one of the producers verbose. So I'd like to keep this configurable per producer.

Reply:
You bring up a good point.  We should be able to set the log level at the processor level. Totally fine to leave as is then.